GA/constraints.py

```
@@ -0,0 +1,546 @@
+            power.append(self._calculate_power(analysis_outputs[i], Lref))
+        
+
+        # Compute the inequality constraints and write them to out["G"]
+        # Rounds the constraint values to 5 decimal figures to match the number of sigfigs given by the MTFLOW outputs to avoid rounding errors.
+        computed_ineq_constraints = []
+        if ineq_constraints:
+            for i, outputs in enumerate(analysis_outputs):
+                self.ref_thrust = config.T_ref_constr[i]
+                self.ref_power = config.P_ref_constr[i]
+                self.oper = self.multi_oper[i]
+                for j, constraint in enumerate(ineq_constraints):
```

Ruff (0.11.9)

Loop control variable `j` not used within loop body

Rename unused `j` to `_j`

(B007)